client/landing/gutenboarding/onboarding-block/create-site-error/index.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { ExternalLink } from '@wordpress/components';
+import React, { FunctionComponent, useEffect } from 'react';
+import { useDispatch } from '@wordpress/data';
+import { useI18n } from '@automattic/react-i18n';
+
+/**
+ * Internal dependencies
+ */
+import Link from '../../components/link';
+import { localizeUrl } from 'lib/i18n-utils';
+import { SITE_STORE } from '../../stores/site';
+import { Title, SubTitle } from '../../components/titles';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	linkTo: string;
+}
+
+const CreateSiteError: FunctionComponent< Props > = ( { linkTo } ) => {
+	const { __ } = useI18n();
+
+	const { resetNewSiteFailed } = useDispatch( SITE_STORE );
+
+	useEffect( () => resetNewSiteFailed );
+
+	return (
+		<div className="create-site-error">
+			<Title>{ __( 'Something went wrong…' ) }</Title>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **51** times:
`translate( 'Oops! Something went wrong.' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/create-site-error)